packages/api-types/types/platform.d.ts

```diff
@@ -4738,7 +4778,16 @@ export interface components {
       description?: string
       name: string
       /** @enum {string} */
-      type: 'postgres' | 'bigquery' | 'webhook' | 'datadog' | 'elastic' | 'loki'
+      type:
```

Comment:
you'll have to remove these changes as they cause the TS failures and it's waiting on analytics team to implement, have flagged with them already